modules/LinearBoltzmannSolvers/A_LBSSolver/lbs_structs.h

```diff
@@ -2,7 +2,7 @@
 #define LBS_STRUCTS_H
 
 #include "ChiMath/chi_math.h"
-#include "ChiPhysics/PhysicsMaterial/transportxsections/material_property_transportxsections.h"
+#include "ChiPhysics/PhysicsMaterial/MultiGroupXS/multigroup_xs.h"
```

Comment:
So I am expecting this file to only be included in rare circumstances. I can think of only these right now:
- When it is created (lua wrappers), some SimTests
- When an adjoint is created (the .cc in LBS where this happens)

The unfortunate thing here is that the declaration of `class MultiGroupXS` does require that header so we are getting away with it.

The ones in green here need to be changed to the base:
<img width="586" alt="image" src="https://user-images.githubusercontent.com/25042529/222877878-a7924018-6ac2-489e-aa04-adaaa0c84fec.png">